lib/matplotlib/text.py

```diff
@@ -1710,9 +1710,13 @@ def transform(renderer) -> Transform
 
         arrowprops : dict, optional
             The properties used to draw a `.FancyArrowPatch` arrow between the
-            positions *xy* and *xytext*. Note that the edge of the arrow
-            pointing to *xytext* will be centered on the text itself and may
-            not point directly to the coordinates given in *xytext*.
+            positions *xy* and *xytext*.  Defaults to None, i.e. no arrow is
+            drawn.
+
+            For historical reasons there are two different ways of usage, that
```

Comment:
```suggestion
            For historical reasons, there are two different usages that
```

Reply:
I'm not sure that's what I want to say. IMHO there is only one usage, namely specifying the type of arrows, but there are two ways to do it. I was considering using "two different APIs" but thought that is too technical.

Reply:
Maybe: 

```suggestion
            For historical reasons there are two different ways to specify arrows, "simple" and "fancy":
```

and maybe drop the next clause?

Reply:
Followed @jklymak suggestion.